app/controllers/api/storage.php

```diff
@@ -890,9 +887,17 @@
             throw new Exception(Exception::STORAGE_FILE_NOT_FOUND);
         }
 
+        if ((\strpos($request->getAccept(), 'image/webp') === false) && ('webp' === $output)) { // Fallback webp to jpeg when no browser support
+            $output = 'jpg';
+        }
+
+        $inputs = Config::getParam('storage-inputs');
+        $outputs = Config::getParam('storage-outputs');
+        $fileLogos = Config::getParam('storage-logos');
+
         $path = $file->getAttribute('path');
         $type = \strtolower(\pathinfo($path, PATHINFO_EXTENSION));
-        $algorithm = $file->getAttribute('algorithm', 'none');
+        $algorithm = $file->getAttribute('algorithm', COMPRESSION_TYPE_NONE);
```

Comment:
Where are this constants coming from? Can we have them come from the compression class? 
https://github.com/utopia-php/storage/blob/main/src/Storage/Compression/Compression.php

Reply:
@eldadfux These were already defined and used in other places, I'm just using it in places where it was missed. Can we do the refactor in separate PR?

Reply:
Yes, but we should.